src/register.js

```diff
@@ -37,11 +59,26 @@ export default function registerStories(req, fileName, sbInstance, plugins) {
         }
       };
     };
-
+    if(decorators){
+    decorators.forEach((decor) => {
+      storiesOf.addDecorator(decor);
+      });
+    }
     story.knobs ? storiesOf.addDecorator(withKnobs) : false;
 
+    var readmeOptions = storyOptions.readme;
+    var readmeContent = readmeOptions.singleFileComponentBlockEnabled ? componentConfig.default.__docs : componentConfig.default.readme;
```

Comment:
I'm not sure I understand this bit of code. Also, where does `componentConfig.default.__docs` com from?